src/display/editor/tools.js

```diff
@@ -1178,25 +1182,23 @@ class AnnotationEditorUIManager {
   }
 
   #addCopyPasteListeners() {
-    document.addEventListener("copy", this.#boundCopy);
-    document.addEventListener("cut", this.#boundCut);
-    document.addEventListener("paste", this.#boundPaste);
+    const signal = this._signal;
+    document.addEventListener("copy", this.#boundCopy, { signal });
+    document.addEventListener("cut", this.#boundCut, { signal });
+    document.addEventListener("paste", this.#boundPaste, { signal });
   }
 
   #removeCopyPasteListeners() {
-    document.removeEventListener("copy", this.#boundCopy);
-    document.removeEventListener("cut", this.#boundCut);
-    document.removeEventListener("paste", this.#boundPaste);
+    const signal = this._signal;
+    document.removeEventListener("copy", this.#boundCopy, { signal });
+    document.removeEventListener("cut", this.#boundCut, { signal });
+    document.removeEventListener("paste", this.#boundPaste, { signal });
```

Comment:
Given https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#options we shouldn't need `signal` when removing the events.
```suggestion
    document.removeEventListener("copy", this.#boundCopy);
    document.removeEventListener("cut", this.#boundCut);
    document.removeEventListener("paste", this.#boundPaste);
```